client/me/help/help-teaser-button.jsx

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
Should this file go under /help-courses? Or is it planned to be used elsewhere in /help?